certified-connectors/Secib/apiDefinition.swagger.json

```diff
@@ -0,0 +1,755 @@
+                      "title": "Hypertext link",
+                      "x-ms-visibility": "important"
+                    },
+                    "responsable": {
+                      "type": "object",
+                      "properties": {
+                        "utilisateurId": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Identifier of the lead attorney on the case file.",
+                          "title": "Lead attorney's identifier"
+                        },
+                        "siteId": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Identifier of the lead attorney's site.",
+                          "title": "Site's identifier"
+                        },
+                        "nom": {
+                          "type": "string",
+                          "description": "Lastname of the case file's lead attorney.",
+                          "title": "Lead attorney's lastname",
+                          "x-ms-visibility": "important"
+                        },
+                        "prenom": {
+                          "type": "string",
+                          "description": "Firstname of the case file's lead attorney.",
+                          "title": "Lead attorney's firstname",
+                          "x-ms-visibility": "important"
+                        },
+                        "nomComplet": {
+                          "type": "string",
+                          "description": "Fullname of the case file's lead attorney.",
+                          "title": "Lead attorney's fullname"
+                        },
+                        "actif": {
+                          "type": "boolean",
+                          "description": "Is the lead attorney active ?",
+                          "title": "Active state",
+                          "enum": [true, false]
+                        },
+                        "email": {
+                          "type": "string",
+                          "description": "Email of the case file's lead attorney.",
+                          "title": "Lead attorney's email",
+                          "x-ms-visibility": "important"
+                        },
+                        "login": {
+                          "type": "string",
+                          "description": "Login of the case file's lead attorney.",
+                          "title": "Lead attorney's login"
+                        }
+                      },
+                      "description": "Lead attorney of the case file"
+                    }
+                  },
+                  "description": "result"
+                }
+              }
+            }
+          },
+          "310": {
+            "description": "Too Many Redirects",
+            "schema": {}
```

Comment:
@thomasBeckSecib Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response.

Reply:
Now, every response have a specified schema (but i causes warnings in the paconn validate step)